src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -452,7 +455,7 @@ object TezosTypes {
   object Voting {
 
     final case class Vote(value: String) extends AnyVal
-    final case class Proposal(protocols: List[ProtocolId], block: Block)
+    final case class Proposal(protocols: List[(ProtocolId, ProposalSupporters)], block: Block)
```

Comment:
maybe make a type alias for the pair?

Reply:
I'm not sure, it would spell something like `ProposedProtocol`, `ProtocolProposal`?

I see room for confusion because it would be used to define a `Proposal`... and then what would be the proper attribute name for those instead of `protocols`?

How easy would be to mix the two things when trying to read and reason about client code...?